### DISCUSSION TITLE
Abuse - Registered Trademark [🔒|Thanks]

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [vercel/community#5910] [Help] [answered]
I have a good faith belief that use of the trademark described above is not authorized by the trademark owner, or its agent, or the law. I have taken nominative and other fair uses into consideration.
I swear, under penalty of perjury, that the information in this notification is accurate and that I am the trademark owner, or am authorized to act on behalf of the owner, of an exclusive right that is allegedly infringed.
_
I request removal of the defamatory made by https://squarecloud.vercel.app about "Square Cloud" (https://squarecloud.app/) company. Is making accusations in mention and use the Square Cloud name in your subdomain. The company did not authorize it and requests the content be removed.
_
Company email: contact@squarecloud.app
Owner's email: joao@squarecloud.app
CNPJ: 51.893.307/0001-08
_
The CNPJ query can be made by the Brazilian Federal Revenue Service: https://solucoes.receita.fazenda.gov.br/servicos/cnpjreva/cnpjreva_solicitacao.asp
_
Images:

![Screenshot_20240223-152425.png](https://github.com/vercel/community/assets/72875404/ff0a2bf6-47ec-413a-9c3c-fae69218a107)

![Screenshot_20240223-152415.png](https://github.com/vercel/community/assets/72875404/2868f2ca-5898-4bfa-9dc2-077e660a823c)
_
Federal Revenue - CNPJ - Square Cloud:

![square_receita.pdf](https://github.com/vercel/community/files/14389055/square_receita.pdf)

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out. I'll share this post with the team, but could you also send this report to [abuse@vercel.com](mailto:abuse@vercel.com) so a ticket will be generated? I want to be sure it is tracked appropriately in our system. 🙇

**Comment by Jeiel0rbit** [comment]
> Thanks for reaching out. I'll share this post with the team, but could you also send this report to [abuse@vercel.com](mailto:abuse@vercel.com) so a ticket will be generated? I want to be sure it is tracked appropriately in our system. 🙇 

Done. Thanks for the guidance. 🙏

**Comment by Jeiel0rbit** [comment]
I am waiting for a response with a case of more than 20 days.

Case number: 00200800

thread::YQU7ehXKIl-026luiEBuKkE::

@amyegan any information about the status or guidance?

**Reply by amyegan** [reply]
Thanks for reaching out @Jetrom17. A response to your case should be in your email inbox now. Please follow up there to continue talking with the Safety Support specialist about your case 🙏
